dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -694,15 +694,13 @@ components:
             - SWL
             - NFL
             - service
-            - cwl
```

Comment:
Head-up @garyluu not sure why these are changing in ParsedInformation

Reply:
Hmm...I think I didn't notice the openapi yaml changed with my last elementcollection list commit. this in the PR looks correct